source/mainnet/net/concepts/concepts-baker.rst

```diff
@@ -20,7 +20,7 @@ A node uses a set of cryptographic keys called baker keys to sign the blocks tha
 Baker account
 -------------
 
-Each account can use a set of baker keys to register a baker. Whenever a baker bakes a valid block that gets included in the chain, a reward is paid to the baker's account after some time. The baker account also receives an award after each :ref:`epoch <glossary-epoch>`. The reward per block is determined by the contents of the block, and is usually lower than the reward that is paid after.
+Each account can use a set of baker keys to register a baker. Whenever a baker bakes a valid block that gets included in the chain, a reward is paid to the baker's account (and the baker pool delegators if they have a pool) at :ref:`pay day<glossary-pay-day>`.
```

Comment:
Perhaps add something to the following effect:
The reward is derived from transaction fees paid for transactions included in the block and its predecessors, as well as from newly-minted CCDs.